www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -766,6 +766,22 @@ The corresponding html:
     </body>
 </html>
 ```
+### Android Quirks
+The default API level in the Cordova Android platform has been upgraded. On an Android 9 device, clear text communication is now disabled by default.
+
+Indicates whether the app intends to use cleartext network traffic, such as cleartext HTTP. The default value for apps that target API level 27 or lower is "true". Apps that target API level 28 or higher default to "false".
+When the attribute is set to "false", for example, HTTP and FTP will refuse the app’s requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected.
```

Comment:
```suggestion
By default HTTP and FTP etc. will refuse the apps requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected. You can learn more about the `android:usesCleartextTraffic` setting in the [documentation for Android developers](https://developer.android.com/guide/topics/manifest/application-element).
```

Sorry for the next suggestions. I always find something while proofreading a second time. I want to make sure this setting is properly explained  and attributed with it's security implications.

Reply:
Is it better to link the Android docs here? Open for opinions.